prijateli_tree/app/routers/games.py

```diff
@@ -185,9 +188,14 @@ def get_previous_answers(
         this_answer = [
             a for a in this_neighbor.answers if a.round == last_round
         ][0]
-        complete_name = (
-            f"{this_neighbor.user.first_name} {this_neighbor.user.last_name}: "
-        )
+
+        # Check if names are hidden
+        if names_hidden:
```

Comment:
Why not just use `game.game_type.names_hidden` instead of creating the variable? Since you're only using it once, there's no need to create a variable to reference it, imo.